openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -100,6 +100,10 @@ public class OpenApiCmd implements BLauncherCmd {
     @CommandLine.Option(names = {"--with-tests"}, hidden = true, description = "Generate test files")
     private boolean includeTestFiles;
 
+    @CommandLine.Option(names = {"--resource-functions"}, hidden = true, description = "Generate client with resource" +
```

Comment:
```suggestion
    @CommandLine.Option(names = {"--client-resources"}, hidden = true, description = "Generate the client using resource functions (instead of using remote functions)" +
```